change/@azure-msal-node-extensions-832ab65b-4ac3-41b1-a9d3-ad58efdc6892.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Given extensions is still in alpha, this should be `prerelease`.
```suggestion
  "type": "prerelease",
```